config/config.go

```diff
@@ -344,6 +343,21 @@ func Application(cfg *Config) (*kingpin.Application, *string) {
 		Envar("PMM_AGENT_DEBUG").BoolVar(&cfg.Debug)
 	app.Flag("trace", "Enable trace output (implies debug) [PMM_AGENT_TRACE]").
 		Envar("PMM_AGENT_TRACE").BoolVar(&cfg.Trace)
+	jsonF := app.Flag("json", "Enable JSON output").Action(func(*kingpin.ParseContext) error {
+		logrus.SetFormatter(&logrus.JSONFormatter{}) // with levels and timestamps always present
+		return nil
+	}).Bool()
+
+	app.Flag("version", "Show application version").Short('v').Action(func(*kingpin.ParseContext) error {
+		if *jsonF {
+			fmt.Println(version.FullInfoJson())
+		} else {
+			fmt.Println(version.FullInfo())
+		}
+		os.Exit(0)
+
+		return nil
```

Comment:
Looks like the return is rudimental here.

Reply:
the function has prototype 
{code}
Action func(*ParseContext) error
{/code}

without it the following error:
```
config/config.go:360:2: missing return at end of function
```